app/scripts/migrations/095.ts

```diff
@@ -0,0 +1,102 @@
+import { cloneDeep, fromPairs, map } from 'lodash';
+import { hasProperty, isObject } from '@metamask/utils';
+import { CHAIN_IDS } from '../../../shared/constants/network';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, any>;
+};
+
+export const version = 95;
+
+/**
+ * This migration will operate the following:
+ *
+ * - Delete `showIncomingTransactions` from `featureFlags` in PreferencesController
+ * - Create a new object under PreferencesController named as `incomingTransactionPreferences`
+ * 1. which will collect all added networks including localhost
+ * 2. then append the test networks
+ * 3. each of them would become a key coming with the value Ture/False from `showIncomingTransactions`
```

Comment:
`* 3. each of them would become a key coming with the value True/False from `showIncomingTransactions``

Reply:
I wonder if point out `showIncomingTransactions` this way to indicate it as saved variable is unclear ?